src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -2003,9 +1996,41 @@ var create_overlay = function (app) {
                 error_url_after_new_tab: true,
                 error_message: null,
                 lightbox: null,
+                prevActiveElement: null,
             }
         },
+        props: {
+            frame_src: String,
+        },
         methods: {
+        },
+        watch: {
+            frame_src: function (newValue) {
+                this.$el.querySelector("iframe").src = newValue;
+            },
+            frame_loading: function (newValue) {
+                if (newValue) {
+                    this.$el.querySelector("iframe").src = this.frame_src;
+                }
+            },
+            frame_shown: function (newValue) {
+                if (newValue) {
+                    this.prevActiveElement = document.activeElement;
+                    var btn = this.$el?.querySelector(".pretix-widget-frame-close a");
+                    this.$nextTick(function () {
+                        btn?.focus();
+                    });
+                } else {
+                    this.prevActiveElement?.focus();
+                }
+            },
+            error_message: function (newValue) {
+                if (newValue) {
```

Comment:
Hmm, wouldn't it make sense to focus the "Continue" button of the error message here? In my test, I wasn't really well able to reach it.